tools/system_libs.py

```diff
@@ -693,16 +693,6 @@ def get_files(self):
 
     ignore += LIBC_SOCKETS
 
-    # individual math files
-    ignore += [
-        'abs.c', 'cos.c', 'cosf.c', 'cosl.c', 'sin.c', 'sinf.c', 'sinl.c',
-        'tan.c', 'tanf.c', 'tanl.c', 'acos.c', 'acosf.c', 'acosl.c', 'asin.c',
-        'asinf.c', 'asinl.c', 'atan.c', 'atanf.c', 'atanl.c', 'atan2.c',
-        'atan2f.c', 'atan2l.c', 'exp.c', 'expf.c', 'expl.c', 'log.c', 'logf.c',
-        'logl.c', 'sqrtl.c', 'round.c', 'roundf.c',
-        'fabsl.c', 'ceill.c', 'floorl.c', 'pow.c', 'powf.c', 'powl.c',
-    ]
```

Comment:
this list has `abs, round, roundf` which are not in the list lower down. is that correct? i guess at worst it could have been a bug before that we use the JS code?

Reply:
Yeah I don't think there is any reason to exclude any of these going forward